docs/jwst/pixel_replace/main.rst

```diff
@@ -0,0 +1,34 @@
+Description
+===========
+
+:Classes: `jwst.pixel_replace.PixelReplaceStep`
+:Alias: pixel_replace
+
+During spectral extraction, pixels flagged as bad are ignored in the summation process.
+If a bad pixel is part of the point-spread function (PSF) at a given wavelength, the
+absence of the signal in the flagged pixel will lead to a divot at that wavelength in
+the extracted spectrum.
+
+To avoid this defect in the 1-D spectrum, this step estimates the flux values of pixels
+flagged as ``DO_NOT_USE`` in 2-D extracted spectra, prior to rectification in the
+``resample_spec`` step. ``pixel_replace`` inserts these estimates into the data array,
+unsets the ``DO_NOT_USE`` flag and sets the ``FLUX_ESTIMATED`` flag for each pixel affected.
+
```

Comment:
It may be semi-obvious to readers that because the `resample_spec` step is mentioned here, this step is only applicable to 2D spectral data (as opposed to IFU), but it might be worth explicitly mentioning that.